src/app/day-calendar/day-calendar.component.ts

```diff
@@ -85,6 +88,7 @@ export class DayCalendarComponent implements OnInit, OnChanges, ControlValueAcce
   currentCalendarMode: ECalendarMode = ECalendarMode.Day;
   monthCalendarConfig: IMonthCalendarConfig;
   _shouldShowCurrent: boolean = true;
+  monthIsSelect: boolean = false;
```

Comment:
I think we should go with a more generic approach here, as we might also be showing a year calendar